benchmarks/deep_learning.py

```diff
@@ -705,7 +705,9 @@ def evaluate_pre_trained_cnn_model(dataset: str, cnn_class: type, config: dict,
         metric_label_prefix="Torch fp32",
     )
 
-    circuit_bitwidth = fhe_module.fhe_circuit.graph.maximum_integer_bit_width()
+    circuit_bitwidth = fhe_module.fhe_circuit.graph.maximum_integer_bit_width(
+        is_encrypted_filter=True
+    )
```

Comment:
So we have to put this is_encrypted_filter = True, everywhere now? Shall we build our own API to simplify this?

e.g. `model.max_bitwidth` ?

Reply:
I guess having our own API makes sense in this case yes.

Reply:
yes if we have no choice I'm definitively in favor of having our own API for this ! It's been quite some time I'm already exhausted of having to remember where to find this info

I see two possible solutions : 
- have a utility function like `get_max_bit_width(fhe_circuit)`. But that means we need to import it each time
- have `model.max_bit_width` like @jfrery suggests. This should probably be a property parameter in the BaseEstimator, before we need to check that the model has been properly compiled. But that means we're adding a new attribute, that we'll need to maintain in the future (I can already see this max bit_width becoming more and more obsolete, so not sure how this solution is relevant in the long term)

not sure which one I prefer for now honestly !

Reply:
Alright so the conclusion is just to go back to the old behavior for now.